### ISSUE TITLE
audio choppiness

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#432]
Originally reported on Google Code with ID 433

```
Describe your system:
 - Operating System (be specific): Kubuntu 10.10
 - Machine type (32-bit or 64-bit): 64
 - Mupen64Plus version: 1.99.4
 - Plugins used: default plugins from source (not extra)

Describe the problem:

when building from source, the make file for <mupen64-ui-console> does not change the
<SHAREDIR> path when a PREFIX is provided. at line 167, i think you are missing:

ifeq ($(SHAREDIR),)
  SHAREDIR := $(PREFIX)/share/mupen64plus
endif

this resolves the issue of mupen not being able to find the ini files during start-up
when compiled without providing explicitly SHAREDIR.

Also, i am having trouble with audio choppiness. from searching the forums it, some
people have said it is an audio pipeline issue while others have said that it is because
of syncing the audio and video.

any ideas?
```

Reported by `benjamyn.ward` on 2011-05-28 16:13:39

**Comment by richard42** [comment]
```
There are multiple possible causes for audio choppiness: too slow CPU, too slow video
card, using a slower core (pure interpreter), audio config parameters.  Please post
your CPU type and speed, and the output of a game run from the console which exhibits
the issue.
```

Reported by `richard42g` on 2011-06-01 03:51:09
- **Labels added**: Component-Mupen64, Type-Defect, Milestone-2.0

**Comment by richard42** [comment]
```
CPU: corei7-950, quad-core HT, 3.07 GHz
     top shows that only ~20% of one of the cpus is begin used, peaking at ~27%

I have attached the output of muppen and my muppen config file.

I turned off kwin and that pretty much fixed it (god i can't believe i didn't try this).
however, when i run wine on project64, i still get 50fps, it's just project64 does
not use openGL so some of the blits which can't get rendered practically crash wine
so I like to use muppen.

hence, i'm not sure it is because of compositing. for some reason with kwin off, muppen
starts using more of the cpu, but i always have plenty of cpu resources available.
with kwin and muppen both running, they still consume <10% of the total according to
top. 

Thank you!
```

Reported by `benjamyn.ward` on 2011-06-01 04:26:47

<hr>
- _Attachment: [mupenOutput.txt](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-433/comment-2/mupenOutput.txt)_
- _Attachment: [mupen64plus.cfg](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-433/comment-2/mupen64plus.cfg)_

**Comment by richard42** [comment]
Reported by `sven@narfation.org` on 2011-07-09 17:43:18

**Comment by richard42** [comment]
```
Seems to be a duplicate of bug 438
```

Reported by `sven@narfation.org` on 2011-09-09 11:20:56
- **Status changed**: `Duplicate`
- **Merged into**: #438
